source/code/source/playfab/PlayFabEventPipeline.cpp

```diff
@@ -267,7 +274,7 @@ namespace PlayFab
             }
             else
             {
-                auto requestBatchPtr = std::shared_ptr<const std::vector<std::shared_ptr<const IPlayFabEmitEventRequest>>>(new std::vector<std::shared_ptr<const IPlayFabEmitEventRequest>>(std::move(foundBatchIterator->second)));
+                auto requestBatchPtr = std::shared_ptr<const std::vector<std::shared_ptr<const IPlayFabEmitEventRequest>>>(new std::vector<std::shared_ptr<const IPlayFabEmitEventRequest>>(foundBatchIterator->second));
```

Comment:
>foundBatchIterator [](start = 188, length = 18)

this is another spot where you are doing a copy instead of a move. is that intentional? I would think we'd want to avoid an entire vector copy